grpc/flatbuffers-js-grpc/src/client.ts

```diff
@@ -0,0 +1,20 @@
+import grpc from 'grpc';
+import { HelloRequest } from './greeter_generated';
+import services from './greeter_grpc';
+
+const flatbuffers = require('flatbuffers').flatbuffers;
+
+function main() {
+    console.log(services);
+    let _server = new services.GreeterClient(`localhost:${3000}`, grpc.credentials.createInsecure());
```

Comment:
Use const instead of let. It might be a good idea to build with --strict. Also, don't interpolate `localhost:${3000}` when you can simply write `localhost:3000`.